quinn/src/incoming_connection.rs

```diff
@@ -0,0 +1,163 @@
+use std::{
+    fmt,
+    future::{Future, IntoFuture},
+    net::{IpAddr, SocketAddr},
+    pin::Pin,
+    task::{Context, Poll},
+};
+
+use bytes::BytesMut;
+use proto::ConnectionError;
+use thiserror::Error;
+
+use crate::{
+    connection::{Connecting, Connection},
+    endpoint::EndpointRef,
+};
+
+/// An incoming connection for which the server has not yet begun its part of the handshake
+#[must_use = "futures/streams/sinks do nothing unless you `.await` or poll them"]
```

Comment:
`Future` has its own `must_use` annotation, so this `must_use` annotation isn't needed. The annotation on other quinn futures predates that Rust feature.

Reply:
Removed.